.github/workflows/security.yml

```diff
@@ -53,10 +53,12 @@ jobs:
 
       - name: Cache Trivy vulnerability database
         if: ${{ steps.db_download.outcome == 'success' }}
-        uses: actions/cache/save@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4.1.2
+        uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4.1.2
         with:
           path: /home/runner/vuln-cache
-          key: trivy-latest-cache
+          key: trivy-cache-${{ github.run_id }}
```

Comment:
What is the default lifetime of those caches? Should we set it to e.g. one week so that GitHub can safely delete them afterwards?

Reply:
The default (and as far as I know immutable) expiry time is of one week already so we should be good :)